x-pack/plugins/security_solution_serverless/public/get_started/welcome_panel/index.tsx

```diff
@@ -84,13 +57,16 @@ const WelcomePanelComponent = ({
                 </EuiTitle>
               }
               description={
-                <span
-                  css={css`
-                    color: ${euiTheme.colors.mediumShade};
-                  `}
-                >
-                  {item?.description?.({ totalActiveSteps, totalStepsLeft })}
-                </span>
+                <>
+                  <span
+                    css={css`
+                      color: ${euiTheme.colors.mediumShade};
+                    `}
+                  >
+                    {item.description && item.description}
+                  </span>
+                  {item.footer && item.footer}
```

Comment:
Instead of adding the `Spacer` at the top of all the footer components that are passed here, we can add always the `margin-top` when we have the footer. So the span-Spacer will no longer be needed and we can remove it.
```suggestion
                  {item.footer && (
                    <span
                      css={css`
                        margin-top: ${euiTheme.size.l};
                      `}
                    >
                      {item.footer}
                    </span>
                  )}
```